src/components/Trade/TradeInputV2.tsx

```diff
@@ -187,25 +308,35 @@ export const TradeInput = () => {
             buySymbol={buyTradeAsset?.asset?.symbol}
             gasFee={gasFee}
             rate={quote?.rate}
+            isLoading={isLoadingFiatRateData || isLoadingTradeQuote}
+            isError={!sellAssetAccountSupported}
           />
-          <ReceiveSummary
-            isLoading={!quote}
-            symbol={buyTradeAsset?.asset?.symbol ?? ''}
-            amount={toCryptoAmountAfterFees.toString()}
-            beforeFees={buyTradeAsset?.amount ?? ''}
-            protocolFee={protocolFee}
-            shapeShiftFee='0'
-            slippage={slippage}
-          />
+          {sellAssetAccountSupported ? (
```

Comment:
update to wallet support vs sellasset

Reply:
Addressed in https://github.com/shapeshift/web/pull/2789/commits/0211c4303224a3fcd18c5c8dab6601ef8154df6f